packages/sales-api-service/src/services/transactions/process-transaction-queue.js

```diff
@@ -54,7 +54,7 @@ export async function processQueue ({ id }) {
     isRenewal,
     isLicenceForYou
   } of transactionRecord.permissions) {
-    const contact = await resolveContactPayload(licensee)
+    const contact = await resolveContactPayload(transactionRecord.permissions[0], licensee)
```

Comment:
You're introducing technical debt here. Have you created a ticket flagging the bug you're introducing for multibuy?

Reply:
https://eaflood.atlassian.net/browse/IWTF-3428